Devservices derby doesn't start because of missing resource bundle

### Describe the bug

According to the [documentation](https://quarkus.io/guides/datasource#dev-services), zero config setup for `Apache derby` should work. But if it is used, there is an error preventing db to start:
```
java.util.MissingResourceException: Can't find bundle for base name org.apache.derby.loc.drda.messages, locale en
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2055)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1689)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1593)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1556)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:932)
	at org.apache.derby.iapi.tools.i18n.LocalizedResource.setResource(Unknown Source)
```

### Expected behavior

Derby dev service should start.

### Actual behavior

_No response_

### How to Reproduce?

Issue could be reproduced with quarkus-quickstart `spring-data-jpa-quickstart`

1. Clone `https://github.com/quarkusio/quarkus-quickstarts/`
2. Modify `https://github.com/quarkusio/quarkus-quickstarts/blob/main/spring-data-jpa-quickstart/pom.xml#L40` to '<artifactId>quarkus-jdbc-derby</artifactId>'
3. From the `quarkus-quickstarts/spring-data-jpa-quickstart` run the test  (i.e. `mvn package`)
4. Test fails with 
```
Caused by: java.util.MissingResourceException: Can't find bundle for base name org.apache.derby.loc.drda.messages, locale en
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2055)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1689)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1593)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1556)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:932)
	at org.apache.derby.iapi.tools.i18n.LocalizedResource.setResource(Unknown Source)
	at org.apache.derby.iapi.tools.i18n.LocalizedResource.getTextMessage(Unknown Source)
	at org.apache.derby.impl.drda.NetworkServerControlImpl.localizeMessage(Unknown Source)
	at org.apache.derby.impl.drda.NetworkServerControlImpl.consolePropertyMessageWork(Unknown Source)
	at org.apache.derby.impl.drda.NetworkServerControlImpl.consolePropertyMessage(Unknown Source)
	at org.apache.derby.impl.drda.NetworkServerControlImpl.setUpSocket(Unknown Source)
	at org.apache.derby.impl.drda.NetworkServerControlImpl.ping(Unknown Source)
	at org.apache.derby.drda.NetworkServerControl.ping(Unknown Source)
	at io.quarkus.devservices.derby.deployment.DerbyDevServicesProcessor$1.startDatabase(DerbyDevServicesProcessor.java:42)
```





### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

**Workaround**

Resource could be added manually into the app. Add following dependency into the https://github.com/quarkusio/quarkus-quickstarts/blob/main/spring-data-jpa-quickstart/pom.xml

```
        <dependency>
            <groupId>org.apache.derby</groupId>
            <artifactId>derbynet</artifactId>
            <scope>test</scope>
        </dependency>
```

and test is successful.

--- a/extensions/jdbc/jdbc-derby/runtime/pom.xml
+++ b/extensions/jdbc/jdbc-derby/runtime/pom.xml
@@ -41,6 +41,10 @@
                 <configuration>
                     <parentFirstArtifacts>
                         <parentFirstArtifact>org.apache.derby:derbyclient</parentFirstArtifact>
+                        <!-- for Dev Services, see: https://github.com/quarkusio/quarkus/issues/21639 -->
+                        <parentFirstArtifact>org.apache.derby:derby</parentFirstArtifact>
+                        <parentFirstArtifact>org.apache.derby:derbynet</parentFirstArtifact>
+                        <parentFirstArtifact>org.apache.derby:derbytools</parentFirstArtifact>
                     </parentFirstArtifacts>
                 </configuration>
             </plugin>

